test/hierarchical/test_spline.py

```diff
@@ -210,7 +215,7 @@ def _inner_problem_exp():
     n_spline_pars = int(np.ceil(spline_ratio * len(timepoints)))
 
     expected_values = {
-        'fun': 0.0,
+        'fun': np.log(2 * np.pi) * 11 / 2,
```

Comment:
Lots of magic numbers in this file, .e.g `11` here. Are they explained somewhere?

Reply:
True :D yes, it's the number of timepoints/datapoints in this example problem. I added a variable.